### PR TITLE
Add UI elements page

### PR DESCRIPTION
**Pull request by imatwawana**

### CODE REVIEW

**Review by jdbaldry — APPROVED** [review]
Cool! 

Looks great locally thanks to that new table expansion feature of the website!

File: docs/sources/write/style-guide/ui-elements/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+To view the images in this table, click **Expand Table** at the top-right corner of the table.
+{{% /admonition %}}
+
+<!-- prettier-ignore-start -->
+
+| UI element | Description | Image | Notes |
+|------------|-------------|-------|-------|
+| breadcrumb | The area at the top of the screen that displays your location in the application and allows you to navigate between the displayed locations. | ![Breadcrumb](/media/docs/writers-toolkit/ui-elements/screenshot-breadcrumb-10.2.png) | Don't refer to this as the _navigation bar_, _top navigation_, or _header_ except in the case of the dashboard header (see below). |
+| dashboard header | The area at the top of the screen that displays your location in the application and includes dashboard tools. | ![Dashboard header](/media/docs/writers-toolkit/ui-elements/screenshot-dashboard-header-10.2-2.png) |  |
+| dialog box | A box that's appears on top of the main screen of the application where you can enter information or select commands. | ![Dialog box](/media/docs/writers-toolkit/ui-elements/screenshot-dialog-box-10.2.png) | Don't use _modal_ or _dialog_ without _box_. |
```
Comment (JStickler) [inline docs/sources/write/style-guide/ui-elements/index.md:24]:
```suggestion
| dialog box | A box that appears on top of the main screen of the application where you can enter information or select commands. | ![Dialog box](/media/docs/writers-toolkit/ui-elements/screenshot-dialog-box-10.2.png) | Don't use _modal_ or _dialog_ without _box_. |
```

File: docs/sources/write/style-guide/ui-elements/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+| main menu | The navigation that opens on the left side of the screen when you click the menu icon (see below). | ![Main menu](/media/docs/writers-toolkit/ui-elements/screenshot-main-menu-10.2.png) | Don't refer to this as the _navigation menu_ or _primary menu_. |
+| menu icon | The element that opens the main menu | ![Menu icon](/media/docs/writers-toolkit/ui-elements/screenshot-menu-icon-10.2.png) | Don't refer to this as the _hamburger menu_. |
```
Comment (JStickler) [inline docs/sources/write/style-guide/ui-elements/index.md:26]:
```suggestion
| menu icon | The element that opens the main menu. | ![Menu icon](/media/docs/writers-toolkit/ui-elements/screenshot-menu-icon-10.2.png) | Don't refer to this as the _hamburger menu_. |
```

Comment (imatwawana) [reply docs/sources/write/style-guide/ui-elements/index.md:26]: Thank for catches!

**Comment by imatwawana** [comment]
> Cool!
> 
> Looks great locally thanks to that new table expansion feature of the website!

Seriously! Table expand for the win!

**Pull request merged**